test/test_image.py

```diff
@@ -100,14 +100,15 @@ def test_decode_jpeg(img_path, pil_mode, mode):
     assert abs_mean_diff < 2
 
 
+@pytest.mark.parametrize("codec", [("png", "PNG"), ("jpg", "JPEG")])
```

Comment:
Not sure if it'll work but this can hopefully just be 

```suggestion
@pytest.mark.parametrize("codec", ("png", "jpeg"))
```

and then just call `codec.upper()` where necessary?

Reply:
I wanted to keep file extension as `.jpg` but `.jpeg` can work as well probably (thinking about all windows os CI).